services/sky/document_view.h

```diff
@@ -41,14 +35,13 @@ class TextureLayer;
 
 class DocumentView : public blink::ServiceProvider,
                      public blink::SkyViewClient,
-                     public mojo::ViewManagerDelegate,
-                     public mojo::ViewObserver,
+                     public mojo::NativeViewportEventDispatcher,
                      public sky::LayerClient,
                      public sky::LayerHostClient,
                      public ui::GestureConsumer {
  public:
-  DocumentView(mojo::InterfaceRequest<mojo::ServiceProvider> services,
-               mojo::ServiceProviderPtr exported_services,
+  DocumentView(mojo::InterfaceRequest<mojo::ServiceProvider> exported_services,
+               mojo::ServiceProviderPtr imported_services,
```

Comment:
I don't think you need either of these after this patch - we'll need to make sure we know how to expose services between embedded things when we add embedding back to the system but I think leaving these unused message pipes around in the meantime only confuses things.  With this patch all sky needs in order to get up and running are connections it creates by talking to the shell.